# sets maximal velocity and acceleration values along the trajectory
dynamic_params: {
  max_v: 0.5,
  max_a: 0.25
}


